### PR TITLE
TECH locale corrections

### PR DESCRIPTION
**Pull request by userjanedoe**
Corrected extension locale.

### CODE REVIEW

File: src/plib/resources/locales/en-US.php [hunk 1]
```diff
@@ -15,24 +15,24 @@
...
+    'list.domains.installLetsencryptDescription' => 'The Let\'s Encrypt extension is required for SSL/TLS certificates creation.',
     'list.wordpress.nameColumn' => 'Name',
     'list.wordpress.urlColumn' => 'URL',
     'list.wordpress.httpsColumn' => 'HTTPS',
     'list.wordpress.httpsEnableTitle' => 'Secure',
     'list.wordpress.httpsDisableTitle' => 'Insecure',
     'list.wordpress.switchToHttpsButtonTitle' => 'Switch to HTTPS',
     'list.wordpress.switchToHttpsButtonDesc' => '',
-    'list.wordpress.notAllowed' => 'The WordPress Toolkit is not allowed by the license key.',
+    'list.wordpress.notAllowed' => 'The WordPress Toolkit extension is not allowed by the license key.',
```
Comment (vlikhtanskiy) [inline src/plib/resources/locales/en-US.php]: It's not extension for Plesk 12.5. Let's don't change this locale key.

File: src/plib/resources/locales/en-US.php [hunk 2]
```diff
@@ -45,28 +45,28 @@
...
     'controllers.system.aiButton' => 'Install nginx web server and enable HTTP/2',
     'controllers.system.datagridDesc' => 'The Datagrid scanner analyzes your server configuration and compares it to real world results from servers around the world to report reliability and security vulnerabilities.  On top of that, it\'s free.',
     'controllers.system.datagrid' => 'Datagrid reliability and vulnerability scanner',
     'controllers.system.datagridActivate' => 'Activate the Datagrid reliability and vulnerability scanner',
     'controllers.system.datagridInstall' => 'Install the Datagrid reliability and vulnerability scanner',
-    'controllers.system.patchmanDesc' => 'Patchman automatically and safely patches vulnerabilities in CMSs like WordPress, Joomla and Drupal. On top of that, it cleans up malware.',
+    'controllers.system.patchmanDesc' => 'Patchman automatically and safely patches vulnerabilities in CMSs like WordPress, Joomla, and Drupal. On top of that, it cleans up malware.',
```
Comment (vlikhtanskiy) [inline src/plib/resources/locales/en-US.php:69]: Why is comma needed here?

File: src/plib/resources/locales/en-US.php (hunk 1 above)
Comment (userjanedoe) [reply src/plib/resources/locales/en-US.php]: OK, then it will be without "the".

File: src/plib/resources/locales/en-US.php (hunk 2 above)
Comment (userjanedoe) [reply src/plib/resources/locales/en-US.php:69]: This is a rule of "serial comma" in English punctuation: https://en.wikipedia.org/wiki/Serial_comma

**Pull request merged**